Update EIP-4750: Disallow dynamic jumps (#6044)
ethereum/EIPs: EIPS/eip-4750.md

    1. Code section is invalid in case offset points to a position outside of section bounds.
    2. Code section is invalid in case offset points to one of two bytes directly following `CALLF` instruction.

> ### Deprecated instructions

Reviewer: It uses "deprecated" in abstract, "disallowed" in motivation, "deprecated" here. We should choose one, and "disallowed" sounds closer to what it is.

Author: Changed to "disallowed" everywhere

> Dynamic jump instructions `JUMP` (`0x56`) and `JUMPI` (`0x57`) are depreceated and invalid and their opcodes do not have any instruction assigned.

Reviewer 2: ```suggestion
Dynamic jump instructions `JUMP` (`0x56`) and `JUMPI` (`0x57`) are depreceated and invalid and their opcodes are undefined.
```

Not sure this is better, but I like "undefined opcode" and/or "undefined instruction" term.

Reviewer: Undefined sounds fine.
